docs/governance/templates/release.md

```diff
@@ -54,8 +54,8 @@ To deploy Open Match in your Kubernetes cluster run the following commands:
 kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=$(YOUR_KUBERNETES_USER_NAME)
 # Place all Open Match components in their own namespace.
 kubectl create namespace open-match
-# Install Open Match and monitoring services.
-kubectl apply -f https://github.com/googleforgames/open-match/releases/download/v{version}/install.yaml --namespace open-match
+# Install Open Match services.
```

Comment:
What are the implications of changing this here? Only impacts release notes?

Reply:
Yes, only the release notes and the docs in `open-match-doc` repo. The `01-open-match-core.yaml` file already exists so we can just pull it from the GCS bucket.